src/etc/gdb_rust_pretty_printing.py

```diff
@@ -10,8 +10,12 @@
 
 import gdb
 import re
+import sys
 import debugger_pretty_printers_common as rustpp
 
+if sys.version_info.major >= 3:
+    xrange = range
```

Comment:
Could you be sure to add a comment here as well indicating why we're doing this?